### DISCUSSION TITLE
Resize image on upload

### DISCUSSION THREAD

**Discussion by huzzi** [SixLabors/ImageSharp.Web#342] [Q&A] [answered]
Is there way of resizing the image to certain width & height during upload? I'm using the package with Umbraco 12.

Please help.

**Comment by ronaldbarendse** [comment, marked as answer]
You can write your own Umbraco `MediaSavingNotification` handler and use the ImageSharp API to resize any images before it's saved, but you can also use an existing Umbraco package that does exactly this: [ImageResizer](https://marketplace.umbraco.com/package/vrijdagonline.imageresizer).

Note that this type of question also doesn't belong here: I'd suggest reaching out to the [Umbraco community](https://community.umbraco.com/get-involved/#social) or ask it in the [forum](https://our.umbraco.com/forum/) instead.
